gitlabci-local.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
Can you please elaborate more why we need this feature?

Reply:
execute gitlab ci pipelines locally with warp speed (52 seconds instead of 5 minutes)